package.json

```diff
@@ -34,7 +34,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build"
```

Comment:
Do we have to define these two scripts: `predeploy` and `deploy` ? I don't like this, as it breaks the consistency of the code